packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -64,22 +65,36 @@ export class Relayer {
     for (const l1Token of Object.keys(relayableDeposits)) {
       this.logger.debug({ at: "Relayer", message: "Processing relayable deposits for L1Token", l1Token });
       for (const relayableDeposit of relayableDeposits[l1Token]) {
+        // If deposit has a pending relay, then validate its relay params (i.e. any data not included in the deposit
```

Comment:
This check is needed since `speedupRelay` can be called now on disputed relays. If the relay is invalid (i.e. it is disputable), then this bot should ignore it and not speed it up.

I'm actually not sure how we handled Disputed relays previously, as I didn't see any tests for it. Was this an oversight @chrismaree ?

Reply:
great addition @nicholaspai . you are 100% correct that before now the bot actually did not validate the correctness of a relay before speeding it up. the previous implementation just processed the client information and assumed correctness.

the implementation you've got here seems good however I have a few comments on the `isRelayValid` method below.